config/zones/US-NW-NWMT.yaml

```diff
@@ -52,6 +55,9 @@ capacity:
     - datetime: '2023-10-01'
       source: EIA.gov
       value: 452.9
+    - datetime: '2024-01-01'
+      source: EIA.gov
+      value: 763.6
```

Comment:
I would use the aggregated number for NWMT, GWA and WWA as you aggregate generation for those zones. Furthermore, the Clearwater Wind 1 wind farm is in the EIA report wrongly accounted to PSEI. The generation is shown in the NWMT zone. The aggregated number would thus be 1528.1